WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressForm.swift

```diff
@@ -103,7 +105,7 @@ struct EditAddressForm: View {
 
                     Group {
                         TitleAndValueRow(title: Localization.countryField, value: Localization.placeholderSelectOption, selectable: true) {
-                            showCountrySelector = true
+                            viewModel.showCountrySelector = true
```

Comment:
Why do you think this should go in the `ViewModel`? 
As this is just part of navigation don't you think it should reside in the `View`?

I have never done this before so I'm wondering what is the best place! 😅

Reply:
I'm also not sure 😄 But after a second look lets keep it in the `View`, since it's accessed only there.
Updated in bcce2f62d3d783dccb8d1aacca9bf4a102212239.